prime-angular-frontend/src/app/modules/adjudication/shared/components/adjudicator-documents/adjudicator-documents.component.html

```diff
@@ -1,75 +1,65 @@
-<app-adjudication-container [content]="content">
+<div class="">
```

Comment:
Unlike React you don't need a parent element in the template

Reply:
Can't stop won't stop